src/System.Dynamic.Runtime/ref/System.Dynamic.Runtime.csproj

```diff
@@ -7,7 +7,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Dynamic.Runtime.cs" />
-    <Compile Include="System.Dynamic.Runtime.Manual.cs" />
+    <!-- ToDo: Remove this P2P Reference once packages are updated -->
```

Comment:
usually @stephentoub or myself would go across the whole repo every once in a while to remove all of these temporary P2P references, but I can defenitely add an issue to track it.